examples/nginx-access-logs-streaming/scripts/sql/01_create-tables.sql

```diff
@@ -0,0 +1,37 @@
+-- 01_create-tables.sql
+
+CREATE STREAM IF NOT EXISTS nginx_historical_access_log
+(
+	remote_ip ipv4, 
+	rfc1413_ident string,
+	remote_user string,
+	date_time_string string,
+	date_time datetime64 default now64(),
+	http_verb string,
+	path string,
+	http_ver string,
+	status uint32,
+	size uint32,
+	referer string,
+	user_agent string,
+	malicious_request string
+) 
+ENGINE = MergeTree
```

Comment:
Better to remove `ENGINE=..` for Timeplus DDL

Reply:
Fixed.